spec/features/markets/search_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "Search for markets" do
+  it "can link to find markets page" do
+    visit root_path
+    click_on "Find a market near you"
+    expect(current_path).to eq(markets_search_path)
+    expect(page).to have_content("Enter address")
+    expect(page).to have_content("Within")
+  end
+  it "can search for market by address" do
+    json_response = File.read('spec/fixtures/markets.json')
+    latitude = 30.69035
```

Comment:
I don't think this is actually testing for a market search by address. It looks like this is simply getting an index of markets, but the params are not being passed anywhere in the request. Am I missing something?